torch/_dynamo/external_utils.py

```diff
@@ -35,13 +37,13 @@ def is_compiling() -> bool:
         return torch.compiler.is_compiling()
 
 
-def wrap_inline(fn: Callable[..., Any]) -> Callable[..., Any]:
+def wrap_inline(fn: Callable[_P, Any]) -> Callable[_P, Any]:
```

Comment:
Could you also propogate the return type too, yby replace 'Any' with a `_R = TypeVar("_R")`

Reply:
Sure, will do!